website/urls.py

```diff
@@ -20,4 +20,11 @@
     path('api/', include(router.urls)),
     path('api/create_patient/', views.create_patient, name='create_patient'),
     path('upload/', views.upload_file, name='upload_file'),
+    path('create_physician/', views.create_physician, name='create_physician'),
```

Comment:
would prefer paths to include `'api'` (e.g., `api/create_physician`, `api/get_patient/`, etc.) but I guess this is ok